src/interval.jl

```diff
@@ -1,12 +1,13 @@
 """
     Interval{T, L <: Bound, R <: Bound}
 
-An `Interval` represents a non-iterable range or span of values (non-interable because,
-unlike a `StepRange`, no step is defined).
+An `Interval` represents a non-iterable range or span of values (non-interable
+because, unlike a `StepRange`, no step is defined).
```

Comment:
```suggestion
An `Interval` represents a non-iterable range or span of values (non-interable because,
unlike a `StepRange`, no step is defined).
```